payment/controllers.go

```diff
@@ -309,19 +309,26 @@ func GetOrderCreds(service *Service) handlers.AppHandler {
 
 		id := uuid.Must(uuid.FromString(orderID))
 
-		creds, err := service.datastore.GetOrderCreds(id)
+		creds, err := service.datastore.GetOrderCreds(id, false)
 		if err != nil {
 			return handlers.WrapError(err, "Error getting claim", http.StatusBadRequest)
 		}
 
 		if creds == nil {
 			return &handlers.AppError{
-				Message: "Order does not exist",
+				Message: "Credentials do not exist",
 				Code:    http.StatusNotFound,
 				Data:    map[string]interface{}{},
 			}
 		}
 
+		for i := 0; i < len(*creds); i++ {
+			if (*creds)[i].SignedCreds == nil {
+				w.WriteHeader(http.StatusAccepted)
```

Comment:
1. does it make sense to return here instead of breaking and going through the creds being encoded, etc.
2. can we add a test to hit status accepted?

Reply:
https://github.com/brave-intl/bat-go/pull/326/commits/d3fa4e7ca74b8f334b0dcd7cbc2c2f75129d4995

https://github.com/brave-intl/bat-go/pull/326/commits/ea0477afa38e4078bf1513326be9ef7acb5401fd